spec/rails_helper.rb

```diff
@@ -13,6 +13,7 @@
 require 'rspec/rails'
 # Add additional requires below this line. Rails is not loaded until this point!
 require 'webmock/rspec'
+require 'sidekiq/testing'
```

Comment:
Sidekiq is in the gemfile by default, it seems like its spec helpers should be included by default as well